Makefile

```diff
@@ -172,10 +178,32 @@ image-build:
 		$(PUSH) \
 		$(IMAGE_BUILD_EXTRA_OPTS) ./
 
+.PHONY: e2e-docker
+e2e-docker:
+	$(IMAGE_BUILD_CMD) -t $(IMAGE_TAG) \
+		--build-arg BASE_IMAGE=$(BASE_IMAGE) \
+		--build-arg BUILDER_IMAGE=$(BUILDER_IMAGE) \
+		--load \
```

Comment:
what is this for?
Could you just add `--load` to the existing `image-build` command?

Reply:
Maybe, we could do it like `image-push`.

https://github.com/kubernetes-sigs/kueue/blob/d136275b81b9592e7a8c888d6f5ed8af2551ce59/Makefile#L176

Reply:
I ran into this issue with using the --platform with --load parameter. https://github.com/docker/buildx/issues/59

I need the --load because it won't show up in my docker image list unless I load it.

Reply:
I see. 
In that case, we may be able to avoid errors by overwriting `PLATFORMS=linux/amd64`

Reply:
Thanks!  That was a good hint!

Reply:
Added:
```
e2e-docker: PLATFORMS=linux/amd64
e2e-docker: IMAGE_BUILD_EXTRA_OPTS=--load
e2e-docker: image-build
```